*** /sourceforge/docbook/schema/relaxng/dbpool.rng	Fri Dec 13 11:53:45 2002
--- dbpoolx.mod	Fri Dec 27 09:10:39 2002
***************
*** 4504,4520 ****
      <ref name="role.attrib"/>
    </define>
! <!-- Add label and role attributes to table and informaltable -->
    <define name="bodyatt">
-     <ref name="label.attrib"/>
-   </define>
- <!-- Add common attributes to Table, TGroup, TBody, THead, TFoot, Row,
-      EntryTbl, and Entry (and InformalTable element). -->
-   <define name="secur">
      <ref name="common.attrib"/>
      <ref name="tables.role.attrib"/>
    </define>
    <define name="common.table.attribs">
      <ref name="bodyatt"/>
!     <ref name="secur"/>
    </define>
  <!-- Content model for Table. -->
--- 4504,4535 ----
      <ref name="role.attrib"/>
    </define>
! <!-- Add common attributes and the Label attribute to Table and -->
! <!-- InformalTable.                                             -->
    <define name="bodyatt">
      <ref name="common.attrib"/>
+     <ref name="label.attrib"/>
      <ref name="tables.role.attrib"/>
    </define>
    <define name="common.table.attribs">
      <ref name="bodyatt"/>
!   </define>
! <!-- Add common attributes to TGroup, ColSpec, TBody, THead, Row, Entry -->
!   <define name="tbl.tgroup.att">
!     <ref name="common.attrib"/>
!   </define>
!   <define name="tbl.colspec.att">
!     <ref name="common.attrib"/>
!   </define>
!   <define name="tbl.tbody.att">
!     <ref name="common.attrib"/>
!   </define>
!   <define name="tbl.thead.att">
!     <ref name="common.attrib"/>
!   </define>
!   <define name="tbl.row.att">
!     <ref name="common.attrib"/>
!   </define>
!   <define name="tbl.entry.att">
!     <ref name="common.attrib"/>
    </define>
  <!-- Content model for Table. -->
***************
*** 4544,4554 ****
  <!-- Allow either objects or inlines; beware of REs between elements. -->
    <define name="tbl.entry.mdl">
!     <choice>
!       <ref name="para.char.mix"/>
!       <ref name="tabentry.mix"/>
!     </choice>
    </define>
! <!-- Reference CALS Table Model -->
!   <include href="calstbl.rng"/>
  <!--end of table.module-->
  <!-- Note that InformalTable is dependent on some of the entity
--- 4559,4571 ----
  <!-- Allow either objects or inlines; beware of REs between elements. -->
    <define name="tbl.entry.mdl">
!     <zeroOrMore>
!       <choice>
!         <ref name="para.char.mix"/>
!         <ref name="tabentry.mix"/>
!       </choice>
!     </zeroOrMore>
    </define>
! <!-- Reference OASIS Exchange Table Model -->
!   <include href="soextblx.rng"/>
  <!--end of table.module-->
  <!-- Note that InformalTable is dependent on some of the entity
